style.css

```diff
@@ -634,3 +634,508 @@ textarea {
+    padding: 8px;
+    color: #4472e7;
+    text-decoration: none;
+    border-radius: 8px;
+    font-size: large;
+    font-family: Poppins, sans-serif;
+    font-weight: bold;
+    background-color: #fff;
+  }
+
+  .btn {
+    border: 2px solid #4472e7;
+    padding: 8px;
+    color: #4472e7;
+    text-decoration: none;
+    border-radius: 8px;
+    font-size: large;
+    font-family: Poppins, sans-serif;
+    font-weight: bold;
+    background-color: #fff;
+    margin-left: 86px;
+  }
+
+  .btn1 {
+    margin-top: 40px;
+    margin-bottom: 30px;
+    align-self: start;
+    font-weight: 500;
+    font-size: 17px;
+  }
+
+  .btn1 a:focus {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  button:hover {
+    background-color: #6070ff;
+    color: #fff;
+    cursor: pointer;
+  }
+
+  button:disabled {
+    color: #5e6c84;
+    background-color: #e5e5e5;
+  }
+
+  button:active {
+    background-color: #2230d2;
+  }
+
+  .about-container {
+    display: flex;
+    background-color: white;
+    border-top-right-radius: 100px;
+    justify-content: center;
+    width: 100%;
+    flex-direction: row;
+    margin-top: 0;
+  }
+
+  .about-container h2 {
+    position: static;
+    width: 100%;
+    height: 52px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 35px;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    margin: 40px 10px 30px 0;
+    font-size: 35px;
+  }
+
+  .about-content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .menu1 {
+    font-family: Poppins, sans-serif;
+    font-weight: bold;
+    color: #344563;
+  }
+
+  .icons {
+    height: 15px;
+    width: 15px;
+  }
+
+  .hero-links {
+    display: flex;
+    margin-left: 30px;
+    margin-right: 30px;
+    margin-bottom: 10px;
+    background-color: #f6f6f9;
+    border-radius: 8px;
+    padding: 10px;
+  }
+
+  .hero-sub-link {
+    font-family: Poppins, sans-serif;
+    color: #344563;
+    padding-top: 10px;
+    padding-left: 15px;
+    font-style: normal;
+  }
+
+  .contact {
+    width: 100%;
+    background: #6070ff;
+    background-image: url(desktopbg-img.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: 143px 11px;
+    border-top-left-radius: 80px;
+    margin-top: -70px;
+    padding: 103px 12px;
+    display: flex;
+    flex-direction: column;
+    gap: 15px;
+    align-items: center;
+  }
+
+  .contact-cntact-1 #contact-me {
+    color: #fff;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .contact-form-section #contact-messeage {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .contact-1 {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    gap: 12px;
+  }
+
+  footer {
```

Comment:
The contact section is also not as per the Figma template of your choosing. Kindly fix it and then ask for a rereview.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178109806-22480081-213d-4062-b3cb-aed7d6610b1e.png) | ![image](https://user-images.githubusercontent.com/33268927/178109831-de7c55ce-3a4e-46de-a976-bb785f4fd3fd.png) |